package.json

```diff
@@ -28,12 +28,15 @@
     "prepublish": "yarn proto && yarn plasmaabi && yarn build && webpack"
   },
   "dependencies": {
+    "@types/lowdb": "^1.0.5",
     "axios": "^0.18.0",
     "bn.js": "^4.11.8",
     "crypto-browserify": "^3.12.0",
     "debug": "^3.1.0",
     "ethereumjs-util": "^5.2.0",
     "google-protobuf": "3.6",
+    "lodash": "^4.17.11",
```

Comment:
lol, how complicated are the two functions you import from lodash? Is it worth pulling in the entirety of lodash?

Reply:
Good point, wasn't even aware of how big lodash is in the first place